### DISCUSSION TITLE
关于始终获取cf 低延迟ip 的方法

### DISCUSSION THREAD

**Discussion by OPPO9008** [XIU2/CloudflareSpeedTest#178] [提问]
根据这个wiki https://github.com/judge2020/cloudflare-connectivity-test/wiki/Explanation
根据不同套餐，路由到的数据中心也不同，假如我们cname ENTERPRISE套餐的域名，是否能始终获取cf 低延迟ip

**Comment by XIU2** [comment]
并不是说高级套餐下域名分配的 IP 就是国内最快的，我试过很多使用 Cloudflare CDN 的大企业的默认分配 IP，就**延迟、丢包、速度**来说，我联通下测速结果和免费套餐域名默认分配的 IP **差不多**，相比而言还是自选的 IP 延迟更低（能相差 100~150ms）。

``` yaml
# 举几个例子，你可以拿它们和你自选的做一下延迟对比，至少我这边相差 150ms 
# 这是 discord.com 的：
162.159.135.232
162.159.136.232
162.159.137.232
162.159.138.232
162.159.128.233

# 这是一个游戏内的 ns.exitgames.com
172.65.212.118
172.65.221.195
172.65.231.130
172.65.231.227
172.65.234.209

# 这两个例子中的域名，都是只能指向固定的这几个 IP，不能随意指向其他 IP（会无法访问），这几个 IP 是固定分配给它们的（独享）
```

而且 Cloudflare CDN 节点里，亚洲节点不少，但是要么**不直连**（就移动直连香港），要么需要**高级套餐/付费**才能用（如有个固定的日本节点 IP 段），主要还是因为 Cloudflare **无法进入中国市场**（也只能和百度、京东搞搞小合作罢了），因此就没打算针对中国内地优化线路（香港、台湾的话，还能直连香港节点），毕竟针对优化就意味着**提高成本**~
